### ISSUE TITLE
Boss taking no damage from any source of attack

### ISSUE THREAD

**Issue by canscixigori** [nickc01/Inferno-King-Grimm#1]
I used this mod multiple times and it was always working fine before beside the fact that whenever i went up against absolute inferno king grimm and when i died my game just massively froze and i always had to force quit. That could be also due to the quality setting i applied against this fight making the background change colour over time. But now, as i am up against inferno king grimm, after some time i just do no damage against him anymore. I tried the debug mod but that didnd't work either. I hope this issue can be fixed as soon as possible because this mod is really cool otherwise.

**Comment by nickc01** [comment]
Yeah, I'm planning on changing the coloring system in a future update. I've been receiving several reports of it creating people's games, so I'm going to try a different method. Do you have a ModLog or Player Log that you can submit? It should be located at "C:\Users\USERNAME\AppData\LocalLow\Team Cherry\Hollow Knight". It should help me narrow down what the damage issue is and I can try and resolve soon.

**Comment by nickc01** [comment]
I just committed a new version of IKG and WeaverCore to Scarab that should fix the invincibility issue. Let me know if it works
